platform/web-security/src/main/java/org/motechproject/security/constants/EmailConstants.java

```diff
@@ -0,0 +1,51 @@
+package org.motechproject.security.constants;
+
+/**
+ * Utility class for storing constants related to e-mails.
+ */
+public final class EmailConstants {
+
+    //Templates
+    public static final String RESET_MAIL_TEMPLATE = "reset_mail.vm";
+    public static final String ONE_TIME_TOKEN_TEMPLATE = "one_time_token_mail.vm";
+    public static final String PASSWORD_CHANGE_REMINDER_TEMPLATE = "password_change_reminder.vm";
+    public static final String LOGIN_INFORMATION_TEMPLATE = "login_info.vm";
```

Comment:
These can stay like this, but when I commented about the snake case, I meant the event parameter names, lines 15-18